AirportDepartures.playground/Contents.swift

```diff
@@ -58,8 +138,15 @@ import UIKit
 //:     Destination: Los Angeles Airline: Delta Air Lines Flight: KL 6966 Departure Time:  Terminal: 4 Status: Canceled
 //:     Destination: Rochester Airline: Jet Blue Airways Flight: B6 586 Departure Time: 1:26 PM Terminal:  Status: Scheduled
 //:     Destination: Boston Airline: KLM Flight: KL 6966 Departure Time: 1:26 PM Terminal: 4 Status: Scheduled
-
-
+func printDepartures2(departureBoard: DepartureBoard) {
+    for departure in departureBoard.departureFlight {
+        if let unwrappedGate = departure.gate, let unwrappedTerminal = departure.terminal, let unwrappedDepartureTime = departure.departureTime, let unwrappedDestination = departure.destinationCity {
```

Comment:
in situations like this its okay to split each let constant on its own by pressing enter after the comma.

```
if let unwrappedGate = departure.gate, 
       let unwrappedTerminal = departure.terminal,
       let unwrappedDepartureTime = departure.departureTime,
       let unwrappedDestination = departure.destinationCity {
```